torch/csrc/dynamo/framelocals_mapping.cpp

```diff
@@ -26,9 +26,13 @@ FrameLocalsMapping::FrameLocalsMapping(FrameLocalsFrameType* frame)
   PyCodeObject* co = F_CODE(frame);
   _framelocals.resize(co->co_nlocalsplus, nullptr);
 
+#if IS_PYTHON_3_14_PLUS
+  TORCH_CHECK(false, "Python 3.14+ not supported");
```

Comment:
Not a dynamo expert but if 3.14 is not expected to work with dynamo wouldn't it be easier to just version guard on the torch.compile() call with an unsupported error?

Reply:
There is already a guard on torch.compile() that's why you don't see it in this PR (I mentioned in the PR decription).
I added these just to make sure it fails and make it easier for William to find these when he does the actual enablement later.